api/python/cellxgene_census/src/cellxgene_census/experimental/ml/pytorch.py

```diff
@@ -131,34 +131,33 @@ def __next__(self) -> ObsDatum:
         while len(obs) < self.batch_size:
             try:
                 obs_partial, X_partial = self._read_partial_torch_batch(self.batch_size)
-                if X is None:
-                    obs = obs_partial
-                    X = X_partial
-                else:
-                    obs = pd.concat([obs, obs_partial], axis=0)
-                    X = sparse.vstack([X, X_partial])
-
+                obs = pd.concat([obs, obs_partial], axis=0)
+                X = sparse.vstack([X, X_partial])
```

Comment:
Unrelated to PR. During debugging, realized `X is None` was always `False` and that the "first iteration" case was not in fact necessary.